ui/main/src/app/modules/usercard/usercard.component.ts

```diff
@@ -450,6 +451,7 @@ export class UserCardComponent implements OnDestroy, OnInit {
             this.startDateVisible = (userCard.startDateVisible === undefined) ? true : userCard.startDateVisible;
             this.endDateVisible = (userCard.endDateVisible === undefined) ? true : userCard.endDateVisible;
             this.lttdVisible = (userCard.lttdVisible === undefined) ? true : userCard.lttdVisible;
+            this.recipientVisible = (userCard.recipientVisible === undefined) ? true : userCard.recipientVisible;
```

Comment:
It's not specific to `recipientVisible`, but it looks like we're defining the default to be `true` 3 times:
- Line 111 when declaring the field
- Here, it the case where it's a usercard
- Line 463 if it's not a usercard

Reply:
Ok, I'll let the default value of all fields at this place then (lines 449-456), and remove it from other places